Gemfile

```diff
@@ -1,18 +1,18 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.6'
+gem 'rails', '~> 5.0.1'
 gem 'pg', '~> 0.18.4'
 gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
 gem 'active_model_serializers'
 gem 'responders'
 gem 'rack-haproxy_status', '~> 0.8.1'
 
-gem 'sprockets-rails', '~> 3.0.4'
+gem 'sprockets-rails'
```

Comment:
Semantic version is vooral handig omdat je dan snel minor (vaak security) updates kan installeren zonder meteen een major (vaak breaking) update te krijgen.